src/CronJobListBuilder.php

```diff
@@ -37,15 +38,22 @@ public function buildHeader() {
    */
   public function buildRow(EntityInterface $entity) {
     /* @var \Drupal\ultimate_cron\Entity\CronJob $entity */
+    $entry = $entity->loadLatestLogEntry();
+    // Start and end as UNIX timestamps.
+    $start_time = $entry->start_time;
```

Comment:
using temporary variables doesn't seem to gain me, we only use those 1-2 times, and using them from the object is only a bit longer...